examples/prometheus/train_single.py

```diff
@@ -180,64 +200,32 @@ def main():
     val_dataset = Subset(val_dataset, range(2))
 
     ################################## The important part #####################################
-    # ZeusMonitor is used to profile the time and energy consumption of the GPU.
-
-    # EnergyHistogram: Records the energy consumption during specific phases of the program execution
-    # and pushes it to the Prometheus Push Gateway as histogram metrics.
-    energy_histogram = EnergyHistogram(
-        cpu_indices=[0], 
-        gpu_indices=[0], 
-        prometheus_url='http://localhost:9091', 
-        job='training_energy_histogram'
-    )
-
-    # PowerGauge: Monitors real-time power usage of the GPUs and pushes the data to the Prometheus 
-    # Push Gateway as gauge metrics, updated at regular intervals.
-    power_gauge = PowerGauge(
-        gpu_indices=[0], 
-        update_period=2, 
-        prometheus_url='http://localhost:9091', 
-        job='training_power_gauge'
-    )
+    # Histogram to track energy consumption over time
+    energy_histogram = EnergyHistogram(cpu_indices=[0,1], gpu_indices=[0], prometheus_url='http://localhost:9091', job='training_energy_histogram')
+    # Gauge to track power consumption over time
+    power_gauge = PowerGauge(gpu_indices=[0], update_period=2, prometheus_url='http://localhost:9091', job='training_power_gauge')
+    # Counter to track energy consumption over time
+    energy_counter = EnergyCumulativeCounter(cpu_indices=[0,1], gpu_indices=[0], update_period=2, prometheus_url='http://localhost:9091', job='training_energy_counter')
```

Comment:
These lines are very long. They're better off using multiple lines, like before the change.